src/app/page.tsx

```diff
@@ -25,19 +25,19 @@ const root = await client.uploadDirectory(files)
 console.log(root.toString())
 `.trim()
 
-export function NavLink ({href, children}: {href:string, children: React.ReactNode}) {
```

Comment:
next build does not like extraneous exports from a page. it expects a single default export only.